packages/fxa-auth-server/lib/subhub/updates.js

```diff
@@ -99,14 +99,18 @@ class SubHubMessageProcessor {
       }
     }
 
+    // The createdAt in our db always has milliseconds, so we compare and store
+    // this value in milliseconds as this Stripe value is in seconds.
+    const messageCreatedAt = message.eventCreatedAt + 1000;
```

Comment:
Did you mean *plus* 1000 or *times* 1000 here?

Reply:
I definitely meant *times*.

Reply:
ouch.  reviewer fail